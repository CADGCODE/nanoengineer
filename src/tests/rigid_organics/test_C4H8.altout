======= test_C4H8.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= stderr =======
cutover to fine tolerance at 2
== structure comparison ==
error reading atom 0 from test_C4H8.xyzcmp
could not read base positions file from -B<filename>
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_C4H8.mmp  > test_C4H8.xyz
final RMS gradient=2783.471713 after 5 iterations
== structure comparison ==
======= test_C4H8.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_C4H8.mmp
# Output File: test_C4H8.xyz
# Trace File: test_C4H8.trc
# Number of Atoms: 12
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#
0 3551.39 4879.20
1 2472.59 3521.71
2 2786.45 4027.45
3 2783.44 4022.57
4 2783.47 4022.62
5 2783.47 4022.62
# Done: Minimization final rms: 2783.471713, highForce: 4022.619908
======= test_C4H8.xyz =======
12
RMS=2783.471713
C 0.208996 -1.116770 0.267140
C -0.988611 -0.313488 -0.537481
C -0.206512 1.117123 -0.267033
C 0.988044 0.312592 0.540086
H -0.154271 -1.574657 1.192470
H 0.772525 -1.776166 -0.383999
H -1.944310 -0.362649 -0.015459
H -1.017867 -0.574264 -1.587422
H -0.774743 1.777974 0.386436
H 0.153062 1.572077 -1.187666
H 1.944645 0.362522 0.022824
H 1.018042 0.575705 1.594104
