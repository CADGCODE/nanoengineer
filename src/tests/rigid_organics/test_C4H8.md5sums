stdout DF379DB31141A33C855918E1FA6D67F7
stderr 160E6504290AEB20212DA51B283BAFE4
test_C4H8.trc C737B751950C3033BA33665BE671DD40
test_C4H8.xyz 6B902F3F0F7CC5D22A1D21CAB7907025
