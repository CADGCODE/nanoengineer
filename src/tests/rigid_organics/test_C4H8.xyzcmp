12
C4H8.mmp
C 0.195000 -1.054000 0.259000
C -0.931000 -0.294000 -0.500000
C -0.195000 1.054000 -0.251000
C 0.931000 0.294000 0.508000
H -0.135000 -1.573000 1.154000
H 0.754000 -1.769000 -0.354000
H -1.922000 -0.366000 -0.047000
H -1.034000 -0.570000 -1.557000
H -0.755000 1.769000 0.354000
H 0.135000 1.573000 -1.154000
H 1.922000 0.366000 0.055000
H 1.034000 0.570000 1.557000
