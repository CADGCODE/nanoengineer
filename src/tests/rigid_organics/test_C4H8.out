======= test_C4H8.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
cutover to fine tolerance at 0
== structure comparison ==
